<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>896</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_12">
    <item>
     <widget class="QTableWidget" name="recordBrowser">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>0</height>
       </size>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>false</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideLeft</enum>
      </property>
      <property name="gridStyle">
       <enum>Qt::SolidLine</enum>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>50</number>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>120</number>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>60</number>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Специальность</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ВУЗ</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Конкурс </string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>ВУЗ</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QRadioButton" name="university_1">
               <property name="text">
                <string>ВолгГТУ</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="university_2">
               <property name="text">
                <string>ВолГУ</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Наименование специальности</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="specialization"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Конкурс прошлого года</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QDoubleSpinBox" name="competition">
               <property name="decimals">
                <number>1</number>
               </property>
               <property name="maximum">
                <double>15.000000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>человек на место</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Размер оплаты при договорном обучении</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QSpinBox" name="tuitionFees">
               <property name="minimum">
                <number>10000</number>
               </property>
               <property name="maximum">
                <number>30000</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>рублей / год</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Наличие магистратуры</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="magistracy">
             <property name="text">
              <string>Есть</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Дата начала подачи документов</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="startSubmissionDate">
             <property name="dateTime">
              <datetime>
               <hour>0</hour>
               <minute>0</minute>
               <second>0</second>
               <year>2024</year>
               <month>1</month>
               <day>1</day>
              </datetime>
             </property>
             <property name="maximumDate">
              <date>
               <year>2024</year>
               <month>12</month>
               <day>31</day>
              </date>
             </property>
             <property name="minimumDate">
              <date>
               <year>2024</year>
               <month>1</month>
               <day>1</day>
              </date>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Дата окончания подачи документов</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="endSubmissionDate">
             <property name="dateTime">
              <datetime>
               <hour>0</hour>
               <minute>0</minute>
               <second>0</second>
               <year>2024</year>
               <month>1</month>
               <day>2</day>
              </datetime>
             </property>
             <property name="maximumDate">
              <date>
               <year>2025</year>
               <month>1</month>
               <day>30</day>
              </date>
             </property>
             <property name="minimumDate">
              <date>
               <year>2024</year>
               <month>1</month>
               <day>2</day>
              </date>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPushButton" name="saveButton">
            <property name="text">
             <string>Сохранить</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteButton">
            <property name="text">
             <string>Удалить</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="addButton">
            <property name="text">
             <string>Создать</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="fillButton">
            <property name="text">
             <string>Заполнить</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>896</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="newFile"/>
    <addaction name="openFile"/>
    <addaction name="saveFile"/>
    <addaction name="saveAsFile"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="openFile">
   <property name="text">
    <string>Открыть...</string>
   </property>
  </action>
  <action name="newFile">
   <property name="text">
    <string>Создать</string>
   </property>
  </action>
  <action name="saveFile">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="saveAsFile">
   <property name="text">
    <string>Сохранить как...</string>
   </property>
  </action>
  <action name="actionjgjgj">
   <property name="text">
    <string>jgjgj</string>
   </property>
  </action>
  <action name="actionjff">
   <property name="text">
    <string>jff</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
